api/handler/put.go

```diff
@@ -745,7 +742,7 @@ func (h *handler) CreateBucketHandler(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	h.log.Info("bucket is created", zap.Stringer("container_id", bktInfo.CID))
+	h.log.Info("bucket is created", zap.String("reqId", reqInfo.RequestID), zap.Stringer("container_id", bktInfo.CID))
```

Comment:
Let's add log bucket name also. And place this between line 731 and 733. It allows to see created container even if we couldn't enable versioning.

Reply:
> And place this between line 731 and 733. It allows us to see created container even if we couldn't enable versioning.

What about this ?

Reply:
Somehow  missed it. Not the logging call is in the place.